README.md

```diff
@@ -1,16 +1,51 @@
 # Vue Snippets
 
-> A Vue project with simple but nice components.
+Vue projects with simple designs yet powerful features.
 
-## Build Setup
+## Components
 
-``` bash
-# install dependencies
-npm install
+Components i have created:
 
-# serve with hot reload at localhost:8080
-npm run dev
+| Component name | Description                        | Location                  |
+|----------------|------------------------------------|---------------------------|
+| Donation box   | A simple, clean donation component | `src/components/donation` |
 
-# build for production with minification
-npm run build
-```
+## Development and building
+
+#### Getting started
+
+To start, download and install the package manager for nodejs `npm`. This is
+used to download and install dependencies used to compile vue files. These
+dependencies can be downloaded and installed using
+
+	npm install
+    
+
+### Development
+
+To build and run for development use the following:
+
+	npm run dev
+    
+This will host the application at `localhost:8080`
+
+### Deployment
+
+To build for production, use the following command:
+
+	npm run build
+    
+The above command also features minification to help reduce file size
+
+## Liniting
+
+Something to look into?
+
+## Authors
+
+- LucasLeandro1204
```

Comment:
Maybe a table with name, profile link and location?

Reply:
When you say location, are we talking about geographical location?

Reply:
Yes, only city and country, I'm from Florianópolis - Brazil